scripts/pre-commit.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+echo "Running detekt check..."
+OUTPUT="/tmp/detekt-$(date +%s)"
+./gradlew detektAll > $OUTPUT
```

Comment:
Also, I'm wondering, should this be using `&>`? That would redirect stderr to the file as well... I don't remember if gradle outputs errors there...
```suggestion
./gradlew detektAll &> $OUTPUT
```

Reply:
I do see the errors when trying to commit something that detekt complains about so I think this is fine